_el/exercises.md

```diff
@@ -510,72 +258,50 @@ have models of what good feedback should look like.
 >     06)          print('total', total)
 >     07)          tot = tot + int(v)
 >     08)      print('total')
-> 
-> 1. poor variable name
-> 1. unused variable
-> 1. use of undefined variable
-> 1. missing return value
+>
+> 1. Κακή ονομασία μεταβλητής
+> 1. Μη χρησιμοποιημένη μεταβλητή
+> 1. Χρήση μη ορισμένης μεταβλητής
+> 1. Έλλειψη επιστρεφόμενης αξίας
```

Comment:
αξίας-->τιμής

Reply:
Τhanks @pdelias ! Fixed in a new commit.